### ISSUE TITLE
Install issue with macOS

### ISSUE THREAD

**Issue by kmace** [r-dbi/RMariaDB#111]
After `install.packages("RMariaDB")` I get the following error:

```
* installing *source* package ‘RMariaDB’ ...
Using mysql_config
Found mysql_config/mariadb_config cflags and libs!
Using PKG_CFLAGS=-I/usr/local/Cellar/mysql/8.0.12/include/mysql 
Using PKG_LIBS=-L/usr/local/opt/openssl/lib -L/usr/local/Cellar/mysql/8.0.12/lib -lmysqlclient -lssl -lcrypto
** libs
clang++  -I"/usr/local/Cellar/r/3.5.0_1/lib/R/include" -DNDEBUG -I/usr/local/Cellar/mysql/8.0.12/include/mysql  -I"/usr/local/lib/R/3.5/site-library/BH/include" -I"/usr/local/lib/R/3.5/site-library/plogr/include" -I"/usr/local/lib/R/3.5/site-library/Rcpp/include" -I/usr/local/opt/gettext/include -I/usr/local/opt/readline/include -I/usr/local/include   -fPIC  -g -O2  -c DbConnection.cpp -o DbConnection.o
clang++  -I"/usr/local/Cellar/r/3.5.0_1/lib/R/include" -DNDEBUG -I/usr/local/Cellar/mysql/8.0.12/include/mysql  -I"/usr/local/lib/R/3.5/site-library/BH/include" -I"/usr/local/lib/R/3.5/site-library/plogr/include" -I"/usr/local/lib/R/3.5/site-library/Rcpp/include" -I/usr/local/opt/gettext/include -I/usr/local/opt/readline/include -I/usr/local/include   -fPIC  -g -O2  -c DbResult.cpp -o DbResult.o
In file included from DbResult.cpp:4:
In file included from ./MariaResultPrep.h:6:
./MariaBinding.h:14:15: error: use of undeclared identifier 'my_bool'
  std::vector<my_bool> is_null;
              ^
In file included from DbResult.cpp:4:
In file included from ./MariaResultPrep.h:9:
./MariaRow.h:18:15: error: use of undeclared identifier 'my_bool'
  std::vector<my_bool> nulls_, errors_;
              ^
DbResult.cpp:77:38: error: no matching constructor for initialization of 'boost::scoped_ptr<MariaResultImpl>'
  boost::scoped_ptr<MariaResultImpl> res(new MariaResultPrep(this, is_statement));
                                     ^   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/lib/R/3.5/site-library/BH/include/boost/smart_ptr/scoped_ptr.hpp:53:5: note: candidate constructor not viable: no known conversion from 'MariaResultPrep *' to 'const boost::scoped_ptr<MariaResultImpl>' for 1st argument
    scoped_ptr(scoped_ptr const &);
    ^
/usr/local/lib/R/3.5/site-library/BH/include/boost/smart_ptr/scoped_ptr.hpp:65:14: note: candidate constructor not viable: no known conversion from 'MariaResultPrep *' to 'MariaResultImpl *' for 1st argument
    explicit scoped_ptr( T * p = 0 ) BOOST_SP_NOEXCEPT : px( p )
             ^
/usr/local/lib/R/3.5/site-library/BH/include/boost/smart_ptr/scoped_ptr.hpp:74:14: note: candidate constructor not viable: no known conversion from 'MariaResultPrep *' to 'std::auto_ptr<MariaResultImpl>' for 1st argument
    explicit scoped_ptr( std::auto_ptr<T> p ) BOOST_SP_NOEXCEPT : px( p.release() )
             ^
3 errors generated.
make: *** [DbResult.o] Error 1
ERROR: compilation failed for package ‘RMariaDB’
* removing ‘/usr/local/lib/R/3.5/site-library/RMariaDB’
Installation failed: Command failed (1)
Warning message:
In get0(".packageName", env, inherits = FALSE) :
  internal error -3 in R_decompress1
```

**Comment by kmace** [comment]
It seems `my_bool` is not defined?

**Comment by krlmlr** [comment]
Thanks. This seems to be addressed with #107 which I just merged. Can you please try installing from GitHub?

**Comment by kmace** [comment]
#107 fixed it. Thanks.
